nf_core/pipeline-template/nextflow.config

```diff
@@ -31,14 +31,14 @@ params {
 
     // Boilerplate options
     outdir                       = null
-    publish_dir_mode             = 'copy'
+    {% if modules %}publish_dir_mode             = 'copy'{% endif %}
```

Comment:
Might be nicer to split these across lines to keep formatting / readability - should be able to use the Jinja `-` syntax to remove that whitespace on template render.

Needn't be a blocker for merging this though.